src/status_im/ui/screens/wallet/request/views.cljs

```diff
@@ -0,0 +1,65 @@
+(ns status-im.ui.screens.wallet.request.views
+  (:require-macros [status-im.utils.views :as views])
+  (:require
+    [re-frame.core :as re-frame]
+    [status-im.components.react :as react]
+    [status-im.components.qr-code :refer [qr-code]]
+    [status-im.components.toolbar-new.actions :as acttions]
+    [status-im.components.toolbar-new.view :as toolbar]
+    [status-im.components.status-bar :as status-bar]
+    [status-im.ui.screens.wallet.send.styles :as wallet-styles]
+    [status-im.components.icons.vector-icons :as vi]
+    [status-im.ui.screens.wallet.ui.components.views :as components]
+    [status-im.ui.screens.wallet.request.styles :as styles]
+    [status-im.i18n :as i18n]
+    [status-im.utils.platform :as platform]))
+
+(defn toolbar-view []
+  [toolbar/toolbar2 {:style wallet-styles/toolbar :hide-border? true}
+   [toolbar/nav-button (acttions/back-white acttions/default-handler)]
+   [toolbar/content-title {:color :white} (i18n/label :t/request-transaction)]])
+
+(defn send-request []
+  (re-frame/dispatch [:navigate-to-modal
+                      :contact-list-modal
+                      {:handler #(re-frame/dispatch [:wallet-send-request %1])
+                       :action  :request
+                       :params  {:hide-actions? true}}]))
+
+(views/defview request-transaction []
+  (views/letsubs [account [:get-current-account]]
+    [react/keyboard-avoiding-view  {:style wallet-styles/wallet-modal-container}
+     [status-bar/status-bar {:type :wallet}]
+     [toolbar-view]
+     [react/view styles/main-container
+      [react/scroll-view
+       [react/view styles/network-container
```

Comment:
Could we use sub functions to split this in smaller part and make more readable?

Reply:
ok